spec/features/post_show_spec.rb

```diff
@@ -0,0 +1,31 @@
+require 'rails_helper'
+RSpec.feature 'User post show page', type: :feature do
+  before do
+    @user = User.find_by(name: 'Tom')
+    @post = @user.posts.first
+  end
+  scenario 'I can see the posts title.' do
+    visit user_post_path(@user, @post)
+    expect(page).to have_content(@post.title)
+  end
+  scenario 'I can see who wrote the post.' do
+    visit user_post_path(@user, @post)
+    expect(page).to have_content(@user.name)
+  end
+  scenario 'I can see how many comments it has.' do
+    visit user_post_path(@user, @post)
+    expect(page).to have_content(@post.comments_counter)
+  end
+  scenario 'I can see how many likes it has.' do
+    visit user_post_path(@user, @post)
+    expect(page).to have_content(@post.likes_counter)
+  end
+  scenario 'I can see the username of each commentor.' do
+    visit user_post_path(@user, @post)
+    expect(page).to have_content(@post.comments.first.user.name)
+  end
+  scenario 'I can see the comment each commentor left.' do
+    visit user_post_path(@user, @post)
+    expect(page).to have_content(@post.comments.first.text)
+  end
+end
```

Comment:
- Please, let's reevaluate the test to ensure they pass as expected 
![image](https://github.com/ulises2607/Blog-App/assets/59397159/2669d1bf-e336-492a-a6f5-d4490808889e)